app/(legal)/imprint/page.tsx

```diff
@@ -0,0 +1,67 @@
+import { ObfuscateSensibleText } from '@/components/ObfuscateSensibleText';
+import { ReactNode } from 'react';
+
+export default function ImprintPage(): ReactNode {
+    return (
+        <div className="flex flex-col items-center">
+            <h1>Imprint</h1>
```

Comment:
Is there any reason this page has a div that centers its items? The text is left-aligned anyways... only the `h1` is centered, and on the Privacy page it is **not**. (this is the same on the lore-le site)

Reply:
Actually don't know, I think that it had something to do with a layout issue, but I fixed it in another way. So thx, I can remove it doesn't really affect the structure.